source/atlas-cli-changelog.txt

```diff
@@ -14,6 +14,18 @@
    :depth: 1
    :class: singlecol
 
+.. _atlas-cli-1.32.0:
+
+{+atlas-cli+} 1.32.0
+--------------------
+
+Released 22 November 2024
+
+- Adds support for SUSE 15.
+- Accepts new format of Atlas Search index definition on commands.
+- Fixes a bug that caused the :ref:`atlas-deployments-search-indexes-create`
+  command ``--watch`` option to watch the wrong index.
```

Comment:
We also removed a duplicate error message when you're not logged in. It would say that you're not logged in twice.

Not sure if it's worth mentioning